### DISCUSSION TITLE
What is the state of ebooks, DRM, and associated software today?

### DISCUSSION THREAD

**Discussion by criddell** [noDRM/DeDRM_tools#656] [General]
I'm curious about where things stand today. Below are where I think things stand but I would appreciate any corrections.

1. Amazon is using KFX for most new books. 

2. If you have an eink Kindle, you can _probably_ remove the DRM from KFX titles you buy today. Tomorrow, it will probably be broken again but the DeDRM tools get updated quickly and it's likely that DRM will be removable again soon. In other words, it's entirely a cat-and-mouse game now?
 
3. Older Amazon formats are pretty thoroughly cracked and you can remove DRM from any of those older files provided you have the physical device / key information.
 
4. Some KFX workarounds involve getting Amazon to send you the file in an older format, but then you lose the typography improvements you get with the KFX version.
 
5. Apple Books DRM has **not** been cracked.
 
6. Adobe's epub DRM has been cracked and books I buy from stores like Google Play Books and the Kobo bookstore can likely be stripped of DRM and read anywhere.
 
7. Commercial DRM removal software mostly just repackage DeDRM and other open source tools into a user friendly application. They have no extra magic in them.
 
8. There's no reason to not upgrade Calibre any more. The plugin issues of the past have been resolved.

The real solution to this mess is probably legislative. Are there any efforts underway to change the books DRM landscape through law?

**Comment by ElleKayEm** [comment]
If you have an E-ink Kindle, you can use download & transfer to get most books in non-kfx older formats.  Currently, I believe the only way to get newly published books in KFX format with older DRM which can be decrypted is the older Kindle for PC 2.4.0.  There are a few books which are only available with DRM which cannot be removed at this time.  As workarounds are closed, someone may become motivated enough to figure out the latest changes and share it.  Amazon is definitely several steps ahead on this.

Even Adobe's "hardened" DRM can be removed.  But most sources aren't even using that anyway.

With Kobo, you also have Obok for books from their desktop app/ereaders.

5 & 7 Yes, as far as I'm aware.

Yes, DeDRM 10.0.9 (and the alpha release version) works with current calibre.  Generally speaking the time to be cautious is when calibre's version jumps to the next whole number indicating a major change.  noDRM has also tried to keep DeDRM backward compatible with older calibre versions, but this is relatively untested.
